src/components/organisms/dashboard/projectsTable.tsx

```diff
@@ -298,17 +393,39 @@ export const DashboardProjectsTable = () => {
 									</Td>
 
 									<Td className="w-1/3 sm:w-1/6">
-										<div className="flex">
-											<IconButton className="group" onClick={() => downloadProjectExport(id)}>
+										<div className="flex justify-start">
+											{status === DeploymentStateVariant.active ? (
+												<IconButton
+													aria-label={t("buttons.stopDeployment")}
+													className="group size-8 p-1"
+													disabled={status !== DeploymentStateVariant.active}
+													onClick={() => handelDeactivateDeployment(deploymentId)}
+													title={t("buttons.stopDeployment")}
+												>
+													<ActionStoppedIcon className="size-4 fill-white transition group-hover:fill-green-200 group-active:fill-green-800" />
+												</IconButton>
+											) : null}
```

Comment:
To make it look like this:
![image](https://github.com/user-attachments/assets/9db9f4ad-c51a-4804-8c98-b56b93ab6499)

My suggestion is to do the following:
```suggestion
											) : <div className="size-8 p-1" />}
```
But eventually it's your choice.

Reply:
added